src/main/java/MainApp.java

```diff
@@ -0,0 +1,16 @@
+public class MainApp {
+    public static void main(String[] args) {
+        Excavator excavator = new Excavator();
+        Bulldozer bulldozer = new Bulldozer();
+        Truck truck = new Truck();
+
+        Machine[] machine = new Machine[] {excavator, bulldozer, truck};
```

Comment:
Variables for single usage are rarely required
```suggestion
        Machine[] machine = new Machine[] {new Excavator(), new Bulldozer(), new Truck()};
```